### ISSUE TITLE
What about add compatibility icons to this add-on?

### ISSUE THREAD

**Issue by mndtr** [denschub/chrome-mask#17]
### I talking about one of this two nice icons:
![изображение](https://github.com/user-attachments/assets/25e487d7-ad86-40c4-b427-b8beb8044c4b)
![изображение](https://github.com/user-attachments/assets/626f321d-877a-4123-bc9f-bf0a64792c2d)

**Comment by denschub** [comment]
For the first: this is my personal project, not a project I built as part of a Mozilla-push. Most of the work on this addon happens in my free time, so it's not a "by Firefox" addon.

The "Recommended" addons are selected based on a few criteria that are documented on https://support.mozilla.org/en-US/kb/recommended-extensions-program#w_how-are-recommended-extensions-selected - that page also explains how users of an addon can nominate it. I sent the addon team an email, but I made very clear that this is my private project, and this addon has less than 3000 users, so it's not liek this is a big deal. :)

tl;dr: That's not something I can do.
